app/src/main/java/com/evanfuhr/pokemondatabase/data/DataBaseHelper.java

```diff
@@ -245,9 +261,12 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
 // You could return cursors by doing "return myDataBase.query(....)" so it'd be easy
 // to you to create adapters for your views.
 
-    public int getVersionGroupIDByVersionID(int version_id) {
+    public int getVersionGroupIDByVersionID() {
         SQLiteDatabase db = this.getWritableDatabase();
 
+        SharedPreferences settings = myContext.getSharedPreferences(String.valueOf(R.string.gameVersionID), MODE_PRIVATE);
+        int version_id = settings.getInt(String.valueOf(R.string.gameVersionID), 0);
+
         int version_group_id = 1;
 
         String selectQuery = "SELECT " + TABLE_VERSIONS + "." + KEY_VERSION_GROUP_ID +
```

Comment:
hmm? Can we use the "select" constant you created above here?

Reply:
So the SQL constants was one approach I'm thinking about. I'm also considering making a SQL builder class.